Sources/Datadog/Datadog.swift

```diff
@@ -215,20 +214,11 @@ public class Datadog {
         if let rumConfiguration = configuration.rum {
             try RUMMonitor.initialize(in: core, configuration: rumConfiguration)
 
-            telemetry.configuration(
-                mobileVitalsUpdatePeriod: rumConfiguration.vitalsFrequency?.toInt64Milliseconds,
-                sessionSampleRate: Int64(withNoOverflow: rumConfiguration.sessionSampler.samplingRate),
-                telemetrySampleRate: Int64(withNoOverflow: rumConfiguration.telemetrySampler.samplingRate),
-                traceSampleRate: Int64(withNoOverflow: rumConfiguration.tracingSampler.samplingRate),
-                trackBackgroundEvents: rumConfiguration.backgroundEventTrackingEnabled,
-                trackFrustrations: rumConfiguration.frustrationTrackingEnabled,
-                trackInteractions: rumConfiguration.instrumentation.uiKitRUMUserActionsPredicate != nil,
-                trackLongTask: rumConfiguration.instrumentation.longTaskThreshold != nil,
-                trackNativeLongTasks: rumConfiguration.instrumentation.longTaskThreshold != nil,
-                trackNativeViews: rumConfiguration.instrumentation.uiKitRUMViewsPredicate != nil,
-                trackNetworkRequests: rumConfiguration.firstPartyHosts != nil,
-                useFirstPartyHosts: rumConfiguration.firstPartyHosts.map { !$0.hosts.isEmpty }
-            )
```

Comment:
We don't want to set these anymore?

Reply:
It has been movde to `RUMMonitor.initialize` 👍